src/entities/supportRequest.ts

```diff
@@ -37,6 +39,12 @@ export class SupportRequest extends BaseEntity {
   @Column({ nullable: false })
   name: string;
 
+  @Column({ nullable: true })
+  primaryLanguage: string;
+
+  @Column('simple-json')
```

Comment:
Since we've removed the test database we can go ahead and use an actual `jsonb` column type and let Postgres handle the conversion

```suggestion
  @Column('jsonb')
```